Libraries/Opc.Ua.Client/ReverseConnectManager.cs

```diff
@@ -340,21 +340,24 @@ private void CloseHosts()
         {
             lock (m_lock)
             {
-                foreach (var host in m_endpointUrls)
+                if (m_endpointUrls != null)
```

Comment:
Not really needed as it only can happen on double dispose, and then it should better crash.  Maybe a Debug.Assert(endointUrls != null) would be better.

Reply:
ok, removed